src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobWrapper.cs

```diff
@@ -0,0 +1,35 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Reindex.Models
+{
+    /// <summary>
+    /// Class to wrap reindex document returneded from persistence layer
+    /// </summary>
+    public class ReindexJobWrapper
+    {
+        public ReindexJobWrapper(ReindexJobRecord jobRecord, WeakETag eTag)
```

Comment:
Do these models need `[JsonConstructor] protected ReindexJobWrapper ()` for deserialization?